doc/configuration_options.rst

```diff
@@ -452,6 +452,28 @@ The following options are valid in version ``1`` (beside the generic options):
   summary page.
   These images will automatically be added to the artifacts for each build.
 
+* ``show_plots``: a dictionary of lists, where the key is the title of the
+  plot group and the list contains plot definitions comprised of:
+
+  * ``title``: the title of the plot.
+  * ``y_axis_label``: (optional) a label for the y-axis.
+  * ``master_csv_name``: the name of the CSV file in which to aggregate the
+    results on the Jenkins master.
+    It must be unique among all plot instances on the same Jenkins master.
+  * ``style``: the type of plot used to display the data.
```

Comment:
Please enumerate the allowed values.

Same below for `data_type`, `selection_flag`, and `selection_value`.

Reply:
I did this for `style`, `data_type` and `selection_flag`, but `selection_value` is hard. It differs completely based on `selection_flag`, and I'd prefer not to write out each one.

If this isn't sufficient, I'm tempted to just remove the options until we actually want them to be different. All of the use cases for the performance jobs are currently setting the same value for each.

Reply:
Maybe list at least some examples? As a reader I have no idea what is expected here.

Reply:
Added.

Reply:
The text itself looks good. Please start each sentence on a new line.